.github/workflows/test.yml

```diff
@@ -83,3 +83,4 @@ jobs:
       - uses: dtolnay/rust-toolchain@stable
       - run: cargo build --verbose
       - run: cargo test --verbose
+      - run: cargo check --examples
```

Comment:
this is not actually needed since `cargo test` runs `cargo check` on examples by default.